src/services/matchesClient.ts

```diff
@@ -85,6 +89,7 @@ export interface PicksBan {
 // account ID to test with 120525879
 export const getMatches = async (matchesRequested: number, accountId: string): Promise<Matches> => {
   try {
+    const projectConfig: Config = config
     const url = `http://localhost:8888/matches?account_id=${accountId}&matches_requested=${matchesRequested}`
```

Comment:
Add string interpolation to use the config like so:
```
const url = `http://${projectConfig.host}:${projectConfig.port}/matches?account_id=${accountId}&matches_requested=${matchesRequested}`
```

Reply:
done